src/models/opinion.jl

```diff
@@ -0,0 +1,55 @@
+using Random
+
+mutable struct Citizen <: AbstractAgent
+	id::Int
+	pos::Tuple{Int, Int}
+	stabilized::Bool
+	opinion::Array{Int, 1}
+	prev_opinion::Array{Int, 1}
+end
+
+"""
+    opinion(;dims=(10, 10), nopinions=3, levels_per_opinion=4)
+
+Same as in [Opinion spread model](@ref).
+"""
+function opinion(;dims=(10, 10), nopinions=3, levels_per_opinion=4)
+	space = GridSpace(dims, periodic=true, moore=true)
+	properties = Dict(:nopinions=>nopinions) 
+	model = AgentBasedModel(Citizen, space, scheduler=random_activation, properties=properties)
+	for cell in 1:nv(model)
+		add_agent!(cell, model, false, rand(1:levels_per_opinion, nopinions), rand(1:levels_per_opinion, nopinions))
+	end
+	return model
```

Comment:
The `opinion_agent_step!` and a `dummy_model` need to be returned here too, yeah?

Reply:
`dummy_model` is exported by the package. Do we need to return it here?

Reply:
See [here](https://github.com/JuliaDynamics/Agents.jl/blob/b5405d11da8931bf9f6063e4a91ff81219fcea35/src/models/forestfire.jl#L32). We need to return the model and both steps to keep the return parameters of the [model ecosystem](https://juliadynamics.github.io/Agents.jl/stable/models/) the same for all models within it.

Reply:
(but yes, `dummystep`, not `dummy_model`, sorry)